### PR TITLE
ncfp

### PR DESCRIPTION
**Pull request by widdowquinn**
This is a tool for retrieving coding sequences from NCBI, given a set of protein sequences.

See https://ncfp.readthedocs.io/en/latest/ for docs.

The guidelines say to ask @bioconda/core for a review, but there's no such reviewer named.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by widdowquinn** [comment]
By all means squash these commits into one, BTW.

File: recipes/ncfp/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
+
+# Add more build steps here, if they are necessary.
```
Comment (bgruening) [inline recipes/ncfp/build.sh]: @widdowquinn can you please clean up the recipe a little bit by removing the comments.

File: recipes/ncfp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: http://widdowquinn.github.io/ncfp/
+  license: MIT License
```
Comment (bgruening) [inline recipes/ncfp/meta.yaml]: `MIT` is enough

**Comment by mbargull** [comment]
> The guidelines say to ask `@bioconda/core` for a review, but there's no such reviewer named.

That `@bioconda/core` pings all of `core` team, not just a single reviewer.

> By all means squash these commits into one, BTW.

We "Squash and merge" on default 😉.

Thanks for the recipe!

**Comment by widdowquinn** [comment]
You guys are quick! Thanks!

I'll make those changes and issue another PR.

**Comment by mbargull** [comment]
You can (and should) make changes to this PR just by pushing to the branch you created it from.
(If you otherwise mean with "other PR", that you want submit an additional other recipe, then be our guest 😉.)

File: recipes/ncfp/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
+package:
+  name: ncfp
+  version: "0.1.0"
+
+source:
+  fn: ncfp-0.1.0.tar.gz
+  url: https://pypi.python.org/packages/aa/ae/399662df6cb31eb0b76f324557d9cb15657a2ca7753ee54aeb0d5153475f/ncfp-0.1.0.tar.gz
+  md5: 4ddd2be47b932f4043eb01df9ad5d471
```
[the remark this answers is not in the document]
Comment (widdowquinn) [reply recipes/ncfp/meta.yaml]: I was surprised that updating conda-build changed what I'd done from Friday. Using `conda skeleton pypi ncfp` last week set those variables (as was here: <https://github.com/widdowquinn/bioconda-recipes/commit/4718f5>); rebuilding it today didn't. I'm new to `conda` packaging, so I didn't know what's best and took any output that passed the tests ;)

**Comment by widdowquinn** [comment]
That's my lax use of language ;) I'm pushing up to that branch on my fork, just as before.

File: recipes/ncfp/meta.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (mbargull) [reply recipes/ncfp/meta.yaml]: Yeah, that's just the older output from `conda-build 2.1.x` vs the newer ones from `conda-build 3.x`. Sometime in (hopefully) near future we'll switch to `conda-build 3`. For now you just can copy-paste the snippet I posted above.

**Comment by mbargull** [comment]
Thanks for the recipe @widdowquinn

**Pull request merged**